letsencrypt/auth_handler.py

```diff
@@ -103,6 +104,30 @@ def _choose_challenges(self, domains):
             self.dv_c.extend(dom_dv_c)
             self.cont_c.extend(dom_cont_c)
 
+    def _simple_verify(self, achall, response):
+        """Self-verifies a response with the corresponding challenge"""
+        return_value = True
+
+        if isinstance(response, challenges.HTTP01Response):
```

Comment:
I'd add `else` stmt that calls `logger.debug("Self-verify of challenge $class is not implemented, ignoring $challenge_url")